homeassistant/components/lg_soundbar/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow to configure the LG Soundbar integration."""
+import temescal
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_UNIQUE_ID
+
+from .const import DEFAULT_HOST, DEFAULT_PORT, DOMAIN
+
+DATA_SCHEMA = {
+    vol.Required(CONF_HOST, default=DEFAULT_HOST): str,
+    vol.Required(CONF_PORT, default=DEFAULT_PORT): vol.Coerce(int),
```

Comment:
Why allow configuring the port, isn't it always the same port ?

Reply:
this can not be confirmed, in #23752 the request was to alow user defined port.

Reply:
can you link me the comment about the port?

Reply:
https://github.com/home-assistant/core/issues/23752#issue-441580935,

"It would be nice if there would be an option to configure the soundbar manually with ip/port"

Reply:
@balloob are we happy to resolve this comment as setting the port via config was requested

Reply:
Are there any reasons to make the port configurable? Won't it always be the same port? Unless LG changes something in the future of course.

I personally think it would be enough with IP, but if there are any cases where the port changes, it should be configurable. I think that comment was made because of the lack of manual setup, which this PR fixes.

Reply:
i can see 2 resons to have the port as an option
1: there is no way for me to confirm that the port is the same on all LG soundbars
2: in https://github.com/home-assistant/core/issues/23752 the request mentions that they have Hassio and the soundbar are both on different subnet, in that case they could have any port defined correct?